### PR TITLE
New version: AIMP.AIMP version 5.30.2541

### PR DESCRIPTION
**Pull request by soredake**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/149994)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Installer https://aimp.ru/files/windows/builds/aimp_5.30.2541_w64-inst.exe
 Scan request id: '67fab941-738a-4c6c-845a-ae2433cc04ba'
 Error information: BlockingDetectionFound
 Installer https://aimp.ru/files/windows/builds/aimp_5.30.2541_w32-ints.exe
 Scan request id: '7058e300-06b2-442d-995f-a95acf9f89d5'
 Error information: BlockingDetectionFound
 
 4/18/2024 7:20:12 PM
 
 ****************************************************************************************

(Automated response - build 877.)

**Comment by stephengillie** [comment]
This might be due to a network block of data centers, to prevent automated downloads.

(Automated response - build 877.)

**Comment by soredake** [comment]
I can't give any feedback, it installs fine on real machine.

**Comment by soredake** [comment]
https://github.com/microsoft/winget-pkgs/discussions/158537

**Comment by soredake** [comment]
Can't do anything.

**Comment by stephengillie** [comment]
> @ stephengillie can you please verify & review this manually as it's installs fine on real machine?

Hi @soredake,

That's probably because the real machine has a residential IP address, and this host is probably blocking data center IP addresses, to prevent automated downloads. Unfortunately, this PR hasn't progressed far enough in the pipeline for me to manually complete.

**Comment by soredake** [comment]
Sigh...

File: manifests/a/AIMP/AIMP/5.30.2555/AIMP.AIMP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ReleaseDate: 2024-06-14
+AppsAndFeaturesEntries:
+  - ProductCode: AIMP
+Installers:
+  - Architecture: x86
+    InstallerUrl: https://aimp.ru/files/windows/builds/aimp_5.30.2555_w32.exe
+    InstallerSha256: 0929D88DCFEDFB3AF4ADCDD94312100FF4C3398F1081FDCC5EB362BF6522E6D8
+  - Architecture: x64
+    InstallerUrl: https://aimp.ru/files/windows/builds/aimp_5.30.2555_w64.exe
+    InstallerSha256: 8E22B547FE2AC9E5B053114C8E7B004499E23E3901FB2E631C90BCA6552F1044
```
Comment (stephengillie) [inline manifests/a/AIMP/AIMP/5.30.2555/AIMP.AIMP.installer.yaml:57]:
```suggestion
    InstallerSha256: 14288E305B385D6597AD346A6ACF4A760CF0A7373B7D023ED6E18D8481170C62
  - Architecture: x64
    InstallerUrl: https://aimp.ru/files/windows/builds/aimp_5.30.2555_w64.exe
    InstallerSha256: E36C0E18A7A242AE1E57D2DA48D3A5A0FC669168096E6E94FBC17F62F1DA8636
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @soredake,

> 8/26/2024 9:29:52 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   E36C0E18A7A242AE1E57D2DA48D3A5A0FC669168096E6E94FBC17F62F1DA8636
Expected hash: 8e22b547fe2ac9e5b053114c8e7b004499e23e3901fb2e631c90bca6552f1044
Specified hash doesn't match.
Actual hash:   14288E305B385D6597AD346A6ACF4A760CF0A7373B7D023ED6E18D8481170C62
Expected hash: 0929d88dcfedfb3af4adcdd94312100ff4c3398f1081fdcc5eb362bf6522e6d8

**Pull request closed without merging**